Added/edited new species to list from plant collection

bleds22e opened the pull request.

gmyenni commented:
Ellen, were any of these added species misidentified as species already on the list? Also, where the Genus/species names have changed, can you add the previous names to the Alternate Genus/species columns?

bleds22e commented:
Good questions. Yes, I went ahead and updated all taxonomic/name changes in the document, moving any names to the alternate name columns. If he identified the species as one that was not already in our list, I went ahead and added it to the list. There were some plants that were identified as other things on the list. I put the word doc he sent me onto Dropbox (Plants/DNA/Ferguson_SEINet_edits...). Anything in bold is a change that George made. You should take a look. We really did do pretty well.
